src/ctim/generators/events.clj

```diff
@@ -0,0 +1,25 @@
+(ns ctim.generators.events
```

Comment:
Should generators like this appear in a namespace that doesn't have "test" in the path?